### DISCUSSION TITLE
Change how many spell points are restored on long rest.

### DISCUSSION THREAD

**Discussion by DaviCaamano** [misthero/dnd5e-spellpoints#121] [Q&A]
The reason I like spell points is because it lets me cut how much casters get from a long rest.

I typically have single encounter days so recovering half spell points on long rest helps me balance my spell casters who otherwise have semi-limitless spell slots for all encounters on the normal system.

**Comment by misthero** [comment]
you can customize it, choose Custom Formula for recovery and use `@item.uses.max/2`

![immagine](https://github.com/user-attachments/assets/ecdd7212-1abc-4cd9-bd2e-9796a7477f0d)

**Reply by romaholiday** [reply]
My foundry doesn't show that option. My DND system version is still 3.3.1, is that the reason? If not, how can I put on a custom formula? I really liked this fix since I don't want my players to have full mana after long rest, but I just KNOW that If I don't make it automatic, they won't keep up with the resource.

**Reply by misthero** [reply]
yes, this is in the latest version of foundry/dnd and the spellpoints module.

**Comment by DaviCaamano** [comment]
Awesome
